client/src/main/scala/io/delta/sharing/client/util/ConfUtils.scala

```diff
@@ -59,7 +59,7 @@ object ConfUtils {
 
   val QUERY_TABLE_VERSION_INTERVAL_SECONDS =
     "spark.delta.sharing.streaming.queryTableVersionIntervalSeconds"
-  val QUERY_TABLE_VERSION_INTERVAL_SECONDS_DEFAULT = "30s"
+  val QUERY_TABLE_VERSION_INTERVAL_SECONDS_DEFAULT = "10s"
```

Comment:
Question: Are we comfortable allowing this for everyone from scalability perspective? Or should we limit it to specific customers first?

Reply:
discussed offline, the server should be able to handle this.

Reply:
reverted